src/app/file-browser/components/edit-tags/edit-tags.component.ts

```diff
@@ -1,27 +1,47 @@
-import { Component, OnInit, Input, OnDestroy, OnChanges, DoCheck, HostBinding, ElementRef, Optional, Inject } from '@angular/core';
+/* @format */
+import {
+  Component,
+  OnInit,
+  Input,
+  OnDestroy,
+  OnChanges,
+  DoCheck,
+  HostBinding,
+  ElementRef,
+  Optional,
+  Inject,
+} from '@angular/core';
 import { TagsService } from '@core/services/tags/tags.service';
 import { ItemVO, TagVOData, TagLinkVOData, FolderVO } from '@models';
 import { DataService } from '@shared/services/data/data.service';
 import { Subject, Subscription } from 'rxjs';
-import { HasSubscriptions, unsubscribeAll } from '@shared/utilities/hasSubscriptions';
+import {
+  HasSubscriptions,
+  unsubscribeAll,
+} from '@shared/utilities/hasSubscriptions';
 import { DataStatus } from '@models/data-status.enum';
 import { ApiService } from '@shared/services/api/api.service';
 import { TagResponse } from '@shared/services/api/tag.repo';
 import { BaseResponse } from '@shared/services/api/base';
 import { MessageService } from '@shared/services/message/message.service';
 import { ngIfScaleAnimation } from '@shared/animations';
-import { DIALOG_DATA, DialogRef } from '@root/app/dialog/dialog.module';
+import { DIALOG_DATA, DialogRef, Dialog } from '@root/app/dialog/dialog.module';
 import { SearchService } from '@search/services/search.service';
 
+export type TagType = 'keyword' | 'customMetadata';
```

Comment:
`keyword` is in anticipation of what's currently called "Tags" being renamed to "Keywords" in the UI. I think at that point "tag" can become a more generic term used in code to refer to the union of keywords and custom metadata (until these concepts eventually separate entirely and that union ceases to be relevant).